latch/types/metadata.py

```diff
@@ -418,6 +421,22 @@ class SnakemakeFileParameter(SnakemakeParameter):
     """
 
 
+@dataclass
+class SnakemakeFileMetadata:
+    path: Optional[Path] = None
+    """
+    The path where the file passed to this parameter will be copied.
+    """
+    config: bool = False
+    """
+    Whether or not the file path is exposed in the Snakemake config
```

Comment:
```suggestion
    If `True`, expose the file in the Snakemake config
```
1. "If `True`" phrasing is from Google docstring style guide
2. active voice